### DISCUSSION TITLE
Dokploy server crashed (data gone), how do I import containers/conf on added server?

### DISCUSSION THREAD

**Discussion by LickABrick** [Dokploy/dokploy#872] [General]
Don't ask me how, because it is a stupid reason. But I lost my Dokploy server. Nothing really imported was hosted on it, but I had added a VPS to it as an additional server.

Does Dokploy automatically import all containers / config from this host if I setup a new Dokploy server and added the server again?

**Comment by Siumauricio** [comment]
No, currently we don't have a way to generate a configuration of everything that is hosted on your dokploy instance there's a feature request, my suggestion to avoid that is to make snapshots with your vps provider, in the case of hetzner you can pay for a backup of your entire instance so you prevent any kind of problem.

**Comment by LickABrick** [comment]
Thats unfortunate. I am supposed to have backups but due to multiple (dumb) circumstances the backup for by dokploy instance is a couple days old and does not include the changes I have done which include the VPS I've added.

Not a big issue, guess I just have to export / import some DB's.
